src/e2e/java/teammates/e2e/pageobjects/InstructorHomePage.java

```diff
@@ -64,13 +64,15 @@ public void copySession(int courseTabIndex, int sessionIndex, CourseAttributes c
     public void publishSessionResults(int courseTabIndex, int sessionIndex) {
         WebElement courseTab = getCourseTab(courseTabIndex);
         click(courseTab.findElement(By.id("btn-results-" + sessionIndex)));
-        clickAndConfirm(courseTab.findElement(By.id("btn-publish-" + sessionIndex)));
+        List<WebElement> publishButtons = browser.driver.findElements(By.id("btn-publish-" + sessionIndex));
+        clickAndConfirm(publishButtons.get(publishButtons.size() - 1));
     }
 
     public void unpublishSessionResults(int courseTabIndex, int sessionIndex) {
         WebElement courseTab = getCourseTab(courseTabIndex);
         click(courseTab.findElement(By.id("btn-results-" + sessionIndex)));
-        clickAndConfirm(courseTab.findElement(By.id("btn-unpublish-" + sessionIndex)));
+        List<WebElement> unpublishButtons = browser.driver.findElements(By.id("btn-unpublish-" + sessionIndex));
+        clickAndConfirm(unpublishButtons.get(unpublishButtons.size() - 1));
```

Comment:
Is this change required? Seems like it does the same thing.

Reply:
Yes, I believe it is necessary. e2e tests [fail](https://github.com/TEAMMATES/teammates/runs/3189369963) without the change.

Previously, the unpublish button was attached to the courseTab, so we can simply use `courseTab.findElement(By.id("btn-unpublish-" + sessionIndex))` to find the correct button.

After the change, the unpublish button is now attached to the html body,  so we have to use `browser.driver.findElements(By.id("btn-unpublish-" + sessionIndex))` to search for the button instead. However, there may be multiple buttons with the same id (in different course tabs), so we have to take the last button in the list (since the dropdown menu is now appended to the end of the html body).